cli/network.go

```diff
@@ -234,3 +236,54 @@ EnableIPV6:   false
 ID:           c33c2646dc8ce9162faa65d17e80582475bbe53dc70ba0dc4def4b71e44551d6
 Internal:     false`
 }
+
+// networkListDescription is used to describe network list command in detail and auto generate command doc.
+var networkListDescription = "List networks in pouchd. "
+
+// NetworkListCommand is used to implement 'network list' command.
+type NetworkListCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkListCommand command.
+func (n *NetworkListCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:     "list [OPTIONS] [NAME]",
```

Comment:
Please remove `[OPTIONS] [NAME]`, since you specified no flag and no args is accpeted.